db/models/api.model.js

```diff
@@ -30,18 +35,26 @@ SELECT
 FROM articles
 LEFT JOIN comments
 ON articles.article_id = comments.article_id
-    GROUP BY articles.article_id
-    ORDER BY articles.created_at DESC;
+    GROUP BY articles.article_id 
 `;
 
   const queryValues = [];
 
+  if(!validSortBy.includes(sort_by) || !validSortQuery.includes(order)){
+    return Promise.reject({status: 400, msg: 'bad request'})
+  }
+
+  else if(sort_by) {
```

Comment:
because you default the sort_by in the parameter you will always end up in this if block - see if everything still works if you remove it and just have the sqlQuery below/out in the open 😄